### DISCUSSION TITLE
meet a problem "failed to get IEP-1559 fees"

### DISCUSSION THREAD

**Discussion by cyhzuishuai** [Cyfrin/foundry-full-course-cu#1759] [Q&A]
when I process command  
forge script script/DeploySimpleStorage.s.sol --rpc-url $SEPOLIA_RPC_URL  --private-key $SEPOLIA_PRIVATE_KEY --broadcast
the terminal answer
 Compiling...
[⠃] Compiling 20 files with Solc 0.8.19
[⠊] Solc 0.8.19 finished in 757.09ms
Compiler run successful!
Script ran successfully.

== Return ==
0: contract SimpleStorage 0x8e4B31F1de330f2fA53D6F2a66F7a8CFD7150587

## Setting up 1 EVM.
Error: 
Failed to get EIP-1559 fees

**Comment by cyhzuishuai** [comment]
have been solve    just foundryup

**Reply by EngrPips** [reply]
Hello @cyhzuishuai, Please use the command below, and let's see how it plays out. 

```js
forge script script/DeploySimpleStorage.s.sol --rpc-url $SEPOLIA_RPC_URL --private-key $SEPOLIA_PRIVATE_KEY --broadcast --legacy
```
